transports/noise/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-noise"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Cryptographic handshake protocol using the noise framework."
-version = "0.43.1"
+version = "0.43.2"
```

Comment:
You'll also have to adjust the version in the root manifest under `workspace.dependencies`.